media/graphql.go

```diff
@@ -1190,21 +1185,19 @@ func (s *sharesResolver) Synch() *stringValueResolver {
 	return synch
 }
 
-type partyIdentifierType int
-
 const (
-	partyIdentifierTypeISNI partyIdentifierType = 0
-	partyIdentifierTypeIPI  partyIdentifierType = 1
-	partyIdentifierTypeDPID partyIdentifierType = 2
+	partyIdentifierTypeISNI string = "ISNI"
```

Comment:
Are these constants still used?

Reply:
The code currently use only partyIdentifierTypeIPI and  partyIdentifierTypeDPID (as before this update).
Are you suggesting that partyIdentifierTypeISNI should be removed and add later at the event of using that -if needed?

Reply:
Ok, if they are being used then :+1:.